cmd/bucky/common_sync.go

```diff
@@ -330,14 +349,18 @@ func (ms *metricSyncer) sync(jobc chan *syncJob, srcThrottling map[string]chan s
 					} else if errors.Is(err, errCantReadMetric) {
 						atomic.AddInt64(&ms.stat.copyError, 1)
 						atomic.AddInt64(&ms.stat.nodes[src].copyError, 1)
+						ms.metriclog("Src", job.oldName, src)
```

Comment:
It is not typical to use uppercase for logs. The typical policy in Go is for all logs and error messages to be lowercase.